documentation/MeetingMinutes/MOM-2023-03-24.md

```diff
@@ -0,0 +1,100 @@
+* Theohar (Hari) Konomi (TELUS)
+* Thorsten Lohmar (Ericsson)
+* Toshiyasu Wakayama (KDDI)
+* ...
+
+QoD minutes: [https://github.com/camaraproject/QualityOnDemand/tree/main/documentation/MeetingMinutes](https://github.com/camaraproject/QualityOnDemand/tree/main/documentation/MeetingMinutes)
+
+## Agenda
+
+* Updates
+* PR [Rename properties to new terms #129](https://github.com/camaraproject/QualityOnDemand/pull/129)
+* Issue [#34 Resolving ueAddr ambiguity when it is the UE private address](https://github.com/camaraproject/QualityOnDemand/pull/34)
+* Issue/Discussion [#125 Adds service provider defined QoS profiles](https://github.com/camaraproject/QualityOnDemand/issues/125)
+* Further new and open Issues
+* AOB
+  * Next call
+  * Planning towards release v0.9.0
+
+## Updates
+* New/additional maintainer from Verizon: Mahesh Chapalamadugu [PR 132](https://github.com/camaraproject/QualityOnDemand/pull/132)
+
+## PR [Rename properties to new terms #129](https://github.com/camaraproject/QualityOnDemand/pull/129)
+* Based on the decisions in [#90](https://github.com/camaraproject/QualityOnDemand/issues/90)
+* Akos and Emil offered to do a review before merge 
+* PR will be kept open until Wednesday March 29th 
+* It's important to merge the PR soon as a prerequisite for the next PRs (the changes are across the complete YAML and documentation and merge conflicts would unavoidable)
+
+## Issue [#34 Resolving ueAddr ambiguity when it is the UE private address](https://github.com/camaraproject/QualityOnDemand/pull/34)
+* Emil presented and explained the added scenario(s) within [UE IP address and ports in QoD.PA3.pdf](https://github.com/camaraproject/QualityOnDemand/files/11013180/UE.IP.address.and.ports.in.QoD.PA3.pdf) (attached at issue comment)
+* Proposal is to ask developers to add all attributes they can observe (easily) and not make the minimum set dependent on network configurations
+* Device port(s) might be neither helpful for device identification nor for (current simple) flow description - could be omitted completetly
+* Port(s) should to with the IP addresses into the Device object - Emil will change the proposal accordingly
+* Discussion to be continued within the issue (but should be concluded soon, see "Release Planning" in AOB)
+
+## Issue [#125 Adds service provider defined QoS profiles](https://github.com/camaraproject/QualityOnDemand/issues/125)
+* Randy presented the updated proposal
+* Throughput/bandwidth replaced with neutral term "rate"
+* Q: intended for mobile and/or fixed - A: both
+* Burstrate needs a description/definition of a burst (e.g. duration, percentage of time etc)
+* Discussed if it would make sense to add a "type" of profile ('best effort' vs 'guaranteed') in addition to the attribute which allow to define theses characteristics (e.g. for easier search). Emil will have a closer look and might make a proposal into this direction. Term "GBR" should not be used as telco and mobile specific
```

Comment:
There was also a request to support backwards compatibility with the current QoS Profile enum.

Reply:
Added. It wasn't only about backward compatibility but also to make it easier for developers if they want to use one of the "standard" profiles (and maybe knowing the mapping already out of the documentation).